manifests/b/ByteDance/Feishu/4.8.3/ByteDance.Feishu.locale.zh-CN.yaml

```
@@ -0,0 +1,37 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: ByteDance.Feishu
+PackageVersion: 4.8.3
+PackageLocale: zh-CN
+Publisher: Beijing Feishu Technology Co., Ltd.
+PublisherUrl: https://www.feishu.cn
+PublisherSupportUrl: https://www.feishu.cn/contact
+PrivacyUrl: https://www.feishu.cn/privacy
+Author: 北京飞书科技有限公司
+PackageName: 飞书
```

```suggestion
PackageName: Feishu
```
On a `zh-CN` system does it show `Feishu` in Control Panel? I can't be bothered to check.

---

No, it shows `飞书` instead.